CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,134 @@
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
```

Comment:
```suggestion
As members, contributors, and leaders, we pledge to contribute to create a harassment-free environment within our community for everyone, regardless of age, body
```

Reply:
This code of conduct is the latest version from the widely adopted Contributor Covenant (https://www.contributor-covenant.org/), which I found to be a reasonable semi-standard code of conduct, so just added it to the repo and almost forgot about it.

I now think the proper way to add a code of conduct would be to discuss it in the team first, and then open a new PR to add whatever the agreed version, instead of sneaking it here, so I'll remove it from this PR.